src/lib/y2network/interfaces_collection.rb

```diff
@@ -87,5 +104,129 @@ def ==(other)
     end
 
     alias_method :eql?, :==
+
+  private
+
+    # FIXME: this is only helper when coexisting with old LanItems module
+    # can be used in new API of network-ng for read-only methods. It converts
+    # old LanItems::Items into new Interface objects
+    def lan_items
```

Comment:
name is quite misleading. It is not! lan_items, it is `interfaces`. And if you read it using new object or LanItems should not matter.

Reply:
renamed